packages/html-colors/0.1.0/README.md

```diff
@@ -0,0 +1,486 @@
+# html-colors
+
+<!-- badges: start -->
+[![Project Status: Active – The project has reached a stable, usable
+state and is being actively
+developed.](https://www.repostatus.org/badges/latest/active.svg)](https://www.repostatus.org/#active)
+[![License:
+MIT](https://img.shields.io/badge/license-MIT-green)](https://choosealicense.com/licenses/mit/)
+<!-- badges: end -->
+
+## Overview
+
+`html-colors` is an [Espanso](https://espanso.org) package that allows
+you to insert HTML color codes into your text seamlessly. By using this
+package, shortcuts like `##red` are automatically replaced with their
+corresponding hex color codes (e.g., `#FF0000`). You can also retrieve
+RGB color codes and the color family. It is designed to work in
+conjunction with the
+[`html-utils-package`](https://hub.espanso.org/html-utils-package).
+
+This package is inspired by [Jenny Knuth’s
+gist](https://gist.github.com/jennyknuth/e2d9ee930303d5a5fe8862c6e31819c5).
+A big thank you to Jenny!
+
+## Installation
+
+To install the latest released version of `html-colors`, use [Espanso
+Hub](https://hub.espanso.org/):
+
+``` sh
+espanso install html-colors
+```
+
+To install the development version directly from GitHub, use:
+
+``` sh
+espanso install html-colors --git https://github.com/danielvartan/html-colors/ --external
+```
+
+## Usage
+
+Once installed, `html-colors` will replace the specified shortcuts with
+their corresponding character as you type.
+
+| Trigger                           | Result                             | Color                          |
+|-----------------------------------|------------------------------------|--------------------------------|
+| `##indianred`                     | `#CD5C5C`                          | ![\#CD5C5C](images/CD5C5C.png) |
```

Comment:
Rather than distribute 130 or so image files to users, simply to view in the `README.md`, I was hoping it would be possible to specify the colours in HTML, e.g.:
```
| `##indianred`                     | `#CD5C5C`                          | <div style="width:75px; height:25px; background-color:#CD5C5C;"></div> |
```
but, as you've probably found, GitHub doesn't render HTML inside table cells. I don't know whether the Espanso Hub package pages will have the same problem. None of the existing packages use the technique, but their Markdown is converted to plain HTML, so I would expect the tags to work there.

An alternative option would be to link to your own files at, e.g. https://github.com/danielvartan/espanso-hub/blob/main/packages/html-colors/0.1.0/images/CD5C5C.png etc. and exclude them from the package itself.

Reply:
Good suggestion! I followed your advice and changed the relative paths to GitHub permalinks.

You can see the changes in the following commit: https://github.com/espanso/hub/pull/130/commits/ecee8c9a2f3fb552c3ba47c5a02407dd43a70b4a